crates/revm/benches/burntpix/mod.rs

```diff
@@ -0,0 +1,124 @@
+use crate::burntpix::genesis_alloc::BURNTPIX_ADDRESS;
```

Comment:
can you move this example to `/bins/revm-test`

Reply:
@rakita you mean the whole file? 

I put it here because so i can just import it in the criterion benches [here](https://github.com/bluealloy/revm/pull/1004/files#diff-823244172388c89c67da035e425522a3e72f8cf3102026d0d170a768c8741587R137)

If I move the whole file to `/bins/revm-test` should i just also change its structure and move away from criterion and use `microbench`?

Reply:
Dont use criterion (or microbench), as this is a long running process there is no need to add bench around it, just measure time with a `Instant::now()`